### PR TITLE
adds field for page section widths

### LINKED ISSUE #17: Add "section width" field to page section paragraph type
If we have a field for "section width" we can then set each 1/2/3 column section as 
- full width
- full width with content contained
- content width

which will much easier allow us to have nice layouts for our sites (and we can add more fields like background colour, etc later)

### PR DESCRIPTION
**Pull request by markconroy**
Closes #17

### CODE REVIEW

File: config/install/field.storage.paragraph.lgms_width_of_section.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+entity_type: paragraph
+type: list_string
+settings:
+  allowed_values:
+    -
+      value: full-width
+      label: 'Full width of screen'
+    -
+      value: full-width-contained-content
+      label: 'Full width, with content contained'
```
Comment (msayoung) [inline config/install/field.storage.paragraph.lgms_width_of_section.yml:18]: Not sure I understand this one. Is it wider than the content but not fullscreen? 
Maybe we could call it "Breakout" ?

Comment (markconroy) [reply config/install/field.storage.paragraph.lgms_width_of_section.yml:18]: This one has the container as the full width of the screen, but the content still contained (handy if you want a coloured background on it, but keep the content centered

Comment (msayoung) [reply config/install/field.storage.paragraph.lgms_width_of_section.yml:18]: Got it. Could we also have a breakout. so somewhere between fullscreen and contained?

Comment (markconroy) [reply config/install/field.storage.paragraph.lgms_width_of_section.yml:18]: Great idea!

**Comment by finnlewis** [comment]
Leaving this one for @msayoung and @markconroy

**Pull request merged**